packages/service-provider-server/src/mongodb-patches.ts

```diff
@@ -14,23 +14,23 @@ export function ensureMongoNodeNativePatchesAreApplied(): void {
   alreadyPatched = true;
 }
 
-const poolToConnections = new Map<ConnectionPool, Set<Connection>>();
+const poolToConnections = new Map<any, Set<any>>();
```

Comment:
I feel like this file might be a bit more readable with a `type ConnectionPool = any; type Connection = any;` at the top, just so we know which `any` is which

Reply:
Neat, yes absolutely 👍